gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/api/model/mapper/ApiMigration.java

```diff
@@ -156,6 +160,29 @@ private Endpoint mapEndpoint(io.gravitee.definition.model.Endpoint lb) {
             .build();
     }
 
+    private Analytics mapAnalytics(Logging v2logging) {
+        var loggingEnabled = isLoggingEnabled(v2logging);
+        var v4logging = loggingEnabled ? mapLogging(v2logging) : new io.gravitee.definition.model.v4.analytics.logging.Logging();
```

Comment:
```suggestion
        var v4logging = loggingEnabled ? mapLogging(v2logging) : null;
```

If logging is disabled in V2, I guess we can keep null for the logging configuration